lib/resources/forms.js

```diff
@@ -30,9 +30,15 @@ const excelMimeTypes = {
   xlsx: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
 };
 
-const canReadForm = (auth, form) => (form.state === 'closed'
-  ? auth.canOrReject('form.read', form)
-  : auth.canOrReject(['open_form.read', 'form.read'], form));
+const canReadForm = (auth, form) => {
+  if (form.def.publishedAt === null) {
+    return auth.canOrReject('form.update', form);
+  } else if (form.state === 'closed') {
+    return auth.canOrReject('form.read', form);
```

Comment:
I've changed the order of checks here to make sure that closing a form does not elevate permissions for form-readers.